### PR TITLE
Add code review protocols

### PR DESCRIPTION
**Pull request by stevenyap**
This PR is to add the code review protocol.

### CODE REVIEW

**Comment by stevenyap** [comment]
@WayneTng @jamesfwz Can you review my pull request? Thank you.

**Comment by jamesfwz** [comment]
Can, can you see my comment?

**Comment by stevenyap** [comment]
Give me a :+1: if you guys are okay with it

**Comment by jamesfwz** [comment]
@stevenyap  If you want to test pull and review. I think we should try on gitlab.com, because the layout of github and gitlab are not the same.

File: protocols/code-review/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Billy      | Brian
+Oliver     | Smith
+James      | Robin
+Wayne      | Theo
+
+The assigned coder buddy will be the first coder to approach for code review eg. Billy will find Brian first for code review.
+
+In the event where the coder buddy is unavailable, the coder will find either of the two coders in the next row eg. Billy will find Oliver or Smith if Brian is unavailable. Wayne will find Chacharoti or Yunchang if Theo is unavailable.
+
+In the event where the secondary reviewers are not available, the coder can find anyone to review his code.
```
Comment (WayneTng) [inline protocols/code-review/README.md:17]: It sound weird to use 'in the event' again. it doesn't read smooth

File: protocols/code-review/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Code Review Steps
+- Coder write new code on a new feature branch based off master
+- Coder push feature branch to Gitlab.com and open a merge request
+- Reviewer review/refactor code with coder
+- Reviewer comment 👍  and merge the request
+- Coder refactor code
+- Coder can ask for second review or push to staging
+
+## Review Standard
+The minimum standard we based on to review the code can be found [here](../../README.md#coding-standard).
```
Comment (WayneTng) [inline protocols/code-review/README.md]: Just a suggestion, maybe you can shift the review standard to this page, as I notice that the review standard at that portion seem to be stand alone in that portion, I find that there a better connection if we can link it here.

**Comment by WayneTng** [comment]
:+1:

**Pull request merged**